crates/lib/src/address.rs

```diff
@@ -188,8 +217,39 @@ mod tests {
     use super::*;
     use proptest::prelude::*;
     use spacetimedb_sats::bsatn;
+    use spacetimedb_sats::ser::serde::SerializeWrapper;
     use spacetimedb_sats::GroundSpacetimeType as _;
 
+    #[test]
+    fn address_json_serialization_big_endian() {
+        let addr = Address::from_be_byte_array([0xff, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]);
+
+        let hex = addr.to_hex();
```

Comment:
If the endianness of to_hex flips along with the json serialization, this test won't catch it. Can you also assert that the string starts with FF?